nncf/torch/dynamic_graph/io_handling.py

```diff
@@ -218,7 +220,7 @@ def get_forward_inputs(self, device: str = None) -> Tuple[Tuple[torch.Tensor, ..
         return tuple(args_list), kwargs
 
 
-class ExactInputsInfo(ModelInputInfo):
+class ExampleInputInfo(ModelInputInfo):
     """
```

Comment:
You should still keep the `ExactInputsInfo` in the hierarchy and inherit `ExampleInputInfo` and `LoaderInputInfo` from it, since both those classes *are* providers of exact inputs to the model's forward.

Reply:
done